test/core/client_channel/lb_policy/pick_first_test.cc

```diff
@@ -329,8 +320,7 @@ TEST_F(PickFirstTest, ShufflingDisabledViaEnvVar) {
         BuildUpdate(kAddresses, MakePickFirstConfig(true)), lb_policy_.get());
     EXPECT_TRUE(status.ok()) << status;
     std::vector<absl::string_view> address_order;
-    GetOrderAddressesArePicked(kAddresses, false, &address_order);
-    EXPECT_THAT(address_order, ::testing::ElementsAreArray(kAddresses));
```

Comment:
This expectation is also still needed, for the same reason.

Reply:
Reverted